### PR TITLE
Hello microverse

### PR DESCRIPTION
**Pull request by solog0039**
In this project:
1. linters installed
2. .html and .css files created and tested
3. github work flow created.
4. readme and gitignore files added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,6 @@
 
 # Ada Library Information
 *.ali
+# .gitignore
+node_modules/
+.md
```
Comment (youmari) [inline .gitignore]: - [ ] please try to add the `test.md` to `.gitignore` file  instead of `.md` as required and also try to remove the `test.md`

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @solog0039 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ No linter error 
✔️ Great title and short summary 
✔️ Github flow 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (solog0039) [reply .gitignore]: dear @youmari, I have added test.md to gitignore

**Review by Kandy-Peter — APPROVED** [review]
Hi @solog0039,

# APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏

Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**